common/src/main/proto/TransportMessages.proto

```diff
@@ -107,6 +107,9 @@ enum MessageType {
   REPORT_BARRIER_STAGE_ATTEMPT_FAILURE_RESPONSE = 84;
   SEGMENT_START = 85;
   NOTIFY_REQUIRED_SEGMENT = 86;
+
+  REVISE_LOST_SHUFFLES = 202;
```

Comment:
Why does the number start from 202?

Reply:
To keep compatible with our internal version.